radar/2025-09-29/cht-upgrade-service.md

```diff
@@ -0,0 +1,8 @@
+---
+title:      "cht-upgrade-service"
+ring:       adopt
+quadrant:   platforms-and-aoe-services
+tags:       [deployment-management]
+---
+
+The [CHT Upgrade Service](https://github.com/medic/cht-upgrade-service) provides an interface between CHT Core API and Docker to allow easy startup and one-click upgrades from the CHT Core Admin UI.
```

Comment:
Adding comment here so I don't forget: please add https://github.com/medic/upgrade-service-kubernetes to stop!